tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -681,7 +681,12 @@ EOT
 		pushd attestation-agent
 		source "${HOME}/.cargo/env"
 		target="${ARCH}-unknown-linux-${LIBC}"
-		cargo build --release --target "${target}" --no-default-features --features "${AA_KBC}"
+		if [ "${AA_KBC}" == "eaa_kbc" ]; then
+			AA_RUSTFLAG="-C link-args=-Wl,-rpath,/usr/local/lib/rats-tls"
+			# Currently eaa_kbc module only support this specific platform
+			target="x86_64-unknown-linux-gnu"
```

Comment:
This block will run on all architectures which isn't what we want I think? Can you add a check to only do this if `[ "$ARCH" = x86_64 ]`.